tests/v2/integration/catalogv2/cluster_repo_test.go

```diff
@@ -353,6 +353,8 @@ func AddHelmChart(u *url.URL) error {
 
 // TestOCIRepo tests CREATE, UPDATE, and DELETE operations of OCI ClusterRepo resources
```

Comment:
we don't have to skip these tests. Cherry pick this commit. 

https://github.com/igomez06/rancher/commit/fb27c8240f86e37cd24cfde142641030153c35df